backend/src/api/common.ts

```diff
@@ -51,8 +51,6 @@ export class Common {
   static findRbfTransactions(added: TransactionExtended[], deleted: TransactionExtended[]): { [txid: string]: TransactionExtended } {
     const matches: { [txid: string]: TransactionExtended } = {};
     deleted
-      // The replaced tx must have at least one input with nSequence < maxint-1 (That’s the opt-in)
-      .filter((tx) => tx.vin.some((vin) => vin.sequence < 0xfffffffe))
```

Comment:
What implications does this change has? That we are in practice enabling detection of Full RBF transactions?

Reply:
Yes, otherwise we miss Full RBF replacements

Reply:
Yes, otherwise we miss Full RBF replacements